pkgs/by-name/ws/wsysmon/package.nix

```diff
@@ -18,17 +19,12 @@ stdenv.mkDerivation rec {
     owner = "slyfabi";
     repo = "wsysmon";
     rev = version;
-    sha256 = "sha256-5kfZT+hm064qXoAzi0RdmUqXi8VaXamlbm+FJOrGh3A=";
+    hash = "sha256-5kfZT+hm064qXoAzi0RdmUqXi8VaXamlbm+FJOrGh3A=";
   };
 
   patches = [
-    # Prevent CMake from trying to fetch libraries from GitHub
-    (substituteAll {
-      src = ./dependencies.patch;
-      spdlog_src = spdlog.src;
-    })
-    # Add an installPhase
-    ./install.patch
+    # Dynamically link spdlog, remove dependency on procps, add an installPhase
+    ./fix-deps-and-add-install.patch
```

Comment:
I think it's better to use fetchpatch right? Or is this different from the upstream commit

Reply:
The upstream commit conflicts with our patches, since the CMakeLists.txt file is so tiny. But yes, I should probably insert a commant with a link to that commit.